src/qt/optionsmodel.h

```diff
@@ -2,6 +2,7 @@
 #define OPTIONSMODEL_H
 
 #include <QAbstractListModel>
+#include <QDateEdit>
```

Comment:
```suggestion
#include <QDate>
```
Doesn't look like we need the full declaration for `QDateEdit` in the header.